### PR TITLE
Add localhost tests to Travis.

### PR DESCRIPTION
**Pull request by tswast**
Run the local servers and verify they respond with code 200 to the "/"
path.

Note: bookshelf does not yet run in Travis, since we'll need to set up
client secrets credentials.

### CODE REVIEW

File: java-repo-tools/scripts/test-localhost.sh [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+#
+# This script runs a localhost server Maven plugin and verifies that a request
+# to http://localhost:8080/ does not return an error code.
+
+print_usage () {
+  echo "Usage:" >&2
+  echo "  $0 server-type path/to/project [-- maven arguments]" >&2
+  echo >&2
+  echo "server-type can be any of the following:" >&2
+  echo "  appengine" >&2
```
Comment (lesv) [inline java-repo-tools/scripts/test-localhost.sh]: standard or flex?

File: java-repo-tools/scripts/test-localhost.sh [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  echo "  jetty" >&2
```
Comment (lesv) [inline java-repo-tools/scripts/test-localhost.sh]: gae-flex ??

**Comment by lesv** [comment]
Consider comments, Ok if you choose not to change - LGTM

File: java-repo-tools/scripts/test-localhost.sh (hunk 1 above)
Comment (tswast) [inline java-repo-tools/scripts/test-localhost.sh]: Standard. The names match those of the Maven plugins, but I suppose that could as we add more kinds of samples.

**Pull request merged**